packages/isomorphic/test/webpack.config.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+const assert = require("assert");
+const path = require("path");
+const webpack = require("webpack");
+
+function webpackForTest(testFileName) {
+  const match = testFileName.match(/\/?([^\/]*)\.ts$/);
+  if (!match) {
+    assert(false, "wrong filename:" + testFileName);
+  }
+
+  const test = {
+    mode: "production",
+    cache: true,
+    externals: [
+      {
+        net: "null", // net is used in one spot in only the tests
```

Comment:
Note from before - can you indicate what spot it's used in the tests so that if we remove it, we know that we can remove this line?

Reply:
You are correct.  I have moved more of the clean up to #2466 in there it will explain the place it is used better.